aws-datastore/build.gradle

```diff
@@ -16,6 +16,7 @@
 apply plugin: 'com.android.library'
 apply from: rootProject.file("configuration/checkstyle.gradle")
 apply from: rootProject.file("configuration/publishing.gradle")
+apply plugin: 'kotlin-android'
```

Comment:
Please make sure to add `ktlint` and ensure it is enabled.  Refer to core-kotlin/build.gradle to see how.